app-frontend/src/app/components/common/toggle/toggle.module.js

```diff
@@ -1,10 +1,43 @@
 import angular from 'angular';
-import ToggleComponent from './toggle.component.js';
-import ToggleController from './toggle.controller.js';
+import toggleTpl from './toggle.html';
 
 const ToggleModule = angular.module('components.toggle', []);
 
-ToggleModule.component('rfToggle', ToggleComponent);
+const rfToggle = {
+    templateUrl: toggleTpl,
+    controller: 'ToggleController',
+    transclude: true,
+    bindings: {
+        value: '<',
+        onChange: '&',
+        radio: '<',
+        className: '@',
+        customIcon: '<'
```

Comment:
Why not make this just take the icon's class name instead?

Reply:
The current way allows for style compatibility with the old-toggles in which a pseudo element is used instead of the `i` element. I would be in favor of unifying the structure to always use an `i` element, but I think that would go along with eliminating the `old-toggle` components usage.

Reply:
makes sense